internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -556,15 +558,15 @@ func Builder() *cobra.Command {
 
 	cmd.Flags().StringVar(&opts.ClusterName, flag.ClusterName, "", usage.ClusterName)
 	cmd.Flags().StringVar(&opts.Tier, flag.Tier, DefaultAtlasTier, usage.Tier)
-	cmd.Flags().StringVar(&opts.Provider, flag.Provider, "", usage.Provider)
-	cmd.Flags().StringVarP(&opts.Region, flag.Region, flag.RegionShort, "", usage.Region)
+	cmd.Flags().StringVar(&opts.Provider, flag.Provider, defaultProvider, usage.Provider)
+	cmd.Flags().StringVarP(&opts.Region, flag.Region, flag.RegionShort, defaultRegion, usage.Region)
```

Comment:
a default here has no difference from a user setting the flag causing the command to assume the user set the flag and skipping the question 
TL;DR let's not change the defaults even if they are the default when no flag is set it'll cause the program to ask the user to confirm
```suggestion
	cmd.Flags().StringVar(&opts.Provider, flag.Provider, "", usage.Provider)
	cmd.Flags().StringVarP(&opts.Region, flag.Region, flag.RegionShort, "", usage.Region)
```

Reply:
Oh ok, sorry about that! Removed. Thanks!